vmcnet/train/default_config.py

```diff
@@ -79,6 +79,7 @@ def get_default_config() -> ConfigDict:
                 "distribute": True,
                 "debug_nans": False,  # If true, OVERRIDES config.distribute to be False
                 "initial_seed": 0,
+                "reload_model_state": "",
```

Comment:
Hm conceptually it feels like this should be part of the reload_config (see line 47 above), can we instead add an extra flag there like `use_model_params_only`?

It would make it a little more cumbersome to use this functionality but more consistent with the other reload options:
```
--reload.logdir=LOGDIR_FOR_OLD_RUN
--use_config_file=False
--checkpoint_relative_file_path=checkpoints/BLAH.npz
--use_model_params_only=True
```

To make it a bit simpler for us right now I might also modify the defaults on the boolean variables to support the current use case, i.e. set use_config_file=False and use_model_params_only=True by default and then we only have to specify 
```
--reload.logdir=LOGDIR_FOR_OLD_RUN
--checkpoint_relative_file_path=checkpoints/BLAH.npz
```